dvc/repo/add.py

```diff
@@ -3,52 +3,75 @@
 import os
 import logging
 import colorama
+from six import string_types
 
 from dvc.repo.scm_context import scm_context
 from dvc.stage import Stage
 from dvc.utils import walk_files, LARGE_DIR_SIZE
 from dvc.exceptions import RecursiveAddingWhileUsingFilename
-
+from dvc.progress import Tqdm
 from . import locked
 
-
 logger = logging.getLogger(__name__)
 
 
 @locked
 @scm_context
-def add(repo, target, recursive=False, no_commit=False, fname=None):
+def add(
+    repo,
+    target_list,
```

Comment:
let's just call it
```suggestion
    targets,
```
same as we have in other places.

Reply:
I have to avoid reusing the name `targets` as it's redefined later:

https://github.com/iterative/dvc/blob/8f9369065350a0d6f70858b3ff9e864470af34d9/dvc/repo/add.py#L43

at least one of these vars has to have a different name

Reply:
@casperdcl let's rename that one, so the API is nice and uniform names.